src/vellum/workflows/events/workflow.py

```diff
@@ -36,11 +37,29 @@ def serialize_workflow_definition(self, workflow_definition: Type, _info: Any) -
 
 class _BaseWorkflowEvent(BaseEvent):
     body: _BaseWorkflowExecutionBody
+    _context: Optional["WorkflowContext"] = PrivateAttr(default=None)
```

Comment:
[blocking] I think we'll want to keep this to just the workflow for now, and clean up the `_set_context`'s